esp/src/src/Utility.ts

```diff
@@ -348,6 +348,14 @@ export function alphanumSort(arr, col, caseInsensitive, reverse) {
     }
 }
 
+export function simpleSort(arr, col) {
```

Comment:
Is there no type declarations required in these .ts files @GordonSmith ?

Reply:
TypeScript will auto generate the type files for all TS files.  We may need to set a flag for JS files to see them (if I understand the question).